### ISSUE TITLE
How to set different loss weight for different loss function?

### ISSUE THREAD

**Issue by csxmli2016** [junyanz/CycleGAN#39]
I wonder how to set the loss weight?

**Comment by junyanz** [comment]
Setting hyper-parameters is often a black magic in deep learning models. There is probably no principled way of doing it. In practice, we find that `lambda=10~100` yields good results.
